lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -384,47 +354,74 @@ class CanvasKitRenderer implements Renderer {
     frameTimingsOnBuildFinish();
     frameTimingsOnRasterStart();
 
+    final Rasterizer rasterizer =
+        _getRasterizerForView(view as EngineFlutterView);
+
     rasterizer.draw((scene as LayerScene).layerTree);
     frameTimingsOnRasterFinish();
   }
 
+  final Map<EngineFlutterView, Rasterizer> _rasterizers =
+      <EngineFlutterView, Rasterizer>{};
```

Comment:
When a view is disposed/unregistered, should we remove its associated rasterizer?

Reply:
Yes. I added a TODO